src/lib/components/workflow/dropdown-filter/end-time.svelte

```diff
@@ -1,25 +0,0 @@
-<script lang="ts">
-  import DropdownMenu from '$lib/components/dropdown-menu.svelte';
-  import { translate } from '$lib/i18n/translate';
-  import { workflowSorts } from '$lib/stores/filters';
```

Comment:
> sort is no longer supported

Does this affect all `workflowSort`s or just `StartTime` and `EndTime`?

And can we remove the `workflowSorts` from the `filters` store (or is it needed for backwards compatibility)?

Reply:
Hmm good question. It's all sorts I believe, but we only allow sorting with StartTime/EndTime in the UI unless you manually enter the query in the input. 

If we are getting rid of our sort UI, it makes sense to gut the workflowSorts store. I really don't want to do another backwards compatible check for this small UI feature that can cause issues with a big number of workflows even when enabled so I'm gonna remove it